c2corg_ui/static/js/documentediting.js

```diff
@@ -108,7 +107,7 @@ app.DocumentEditingController = function($scope, $element, $attrs,
    * @type {string}
    * @private
    */
-  this.lang_ = $attrs['appDocumentEditingLang'];
+  this.lang_ = $attrs['appDocumentEditingLang'] || app.utils.readCookie('interface_lang');
```

Comment:
Do you really need to read a cookie for this?
It's better to get the current lang from angular-gettext, see https://github.com/c2corg/v6_ui/blob/9971a4033e75cf98c62a66b998e2d37c188d387e/c2corg_ui/static/js/card.js#L47-L59